### PR TITLE
Bug 1210393: hide the action button after success

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: lib/ui/confirmaction.jsx [hunk 1]
```diff
@@ -69,13 +69,17 @@ var ConfirmAction = React.createClass({
...
-              disabled={this.state.executing && !this.state.resultLoaded}>
-            <bs.Glyphicon glyph={this.props.glyph}/>&nbsp;
-            <span>{this.props.label}</span>
-          </bs.Button>
+          {
+            !(this.state.executing || this.state.result) ? (
+              <bs.Button
+                  onClick={this.executeAction}
+                  bsStyle={this.props.buttonStyle}
+                  hidden={this.state.result}
```
Comment (jonasfj) [inline lib/ui/confirmaction.jsx]: missing `>`

**Comment by jonasfj** [comment]
this is one of the few repositories where test works.. in fact they auto-deploy from master... :)

File: lib/ui/confirmaction.jsx (hunk 1 above)
Comment (djmitche) [inline lib/ui/confirmaction.jsx]: Huh, I wonder how that worked for me.. >>blames grunt<<

**Comment by jonasfj** [comment]
I'm guessing you used `--force` :)

**Comment by jonasfj** [comment]
thanks...

**Pull request merged**